app/views/episodes/new.html.erb

```diff
@@ -4,6 +4,6 @@
 
 <%= render "form", episode: @episode %>
 
-<% if !@podcast.complete && @podcast.episodes.after(Time.now).none? %>
+<% if action_name == "new" && !@podcast.complete && @podcast.episodes.after(Time.now).none? %>
```

Comment:
If you pulled up the new-episode form, submitted with an error (blank title) ... you get re-warned about planning episode drafts.  This only shows it once.